client/js/socket-events/sessions_list.js

```diff
@@ -0,0 +1,31 @@
+"use strict";
+
+const $ = require("jquery");
+const socket = require("../socket");
+const templates = require("../../views");
+
+socket.on("sessions:list", function(data) {
+	data.sort((a, b) => b.lastUse - a.lastUse);
+
+	let html = "";
+	data.forEach((connection) => {
+		if (connection.current) {
+			$("#session-current").html(templates.session(connection));
+			return;
+		}
+
+		html += templates.session(connection);
+	});
+
+	if (html.length === 0) {
+		html = "<em>You are not currently logged in to any other device.</em>";
```

Comment:
This needs to be in a `<p></p>` to respect font size (and also a good idea to match session blocks in case margin or else gets changed).

btw much better idea than what I posted yesterday lol #shame